### ISSUE TITLE
set multiple options for one rule

### ISSUE THREAD

**Issue by artin-phares** [eslint/eslint#9024]
eslint.rc:
```javascript
{
   "rules": {
      "no-trailing-spaces": ["error", {
         "skipBlankLines": true,
         "ignoreComments": true
      }]
   }
}
```

result:
```
λ eslint .
D:\Projects\pvc\.eslintrc:
        Configuration for rule "no-trailing-spaces" is invalid:
        Value "data["0"].ignoreComments" has additional properties.
```

How do I supposed to set multiple options for one rule (eg. [no-trailing-spaces](http://eslint.org/docs/rules/no-trailing-spaces))?  
Setting both options alone - works ok, but I need both.

**Comment by eslintbot** [comment]
Hi @artin-phares, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by artin-phares** [comment]
eslint v4.3.0

**Comment by not-an-aardvark** [comment]
Thanks for the report. However, I'm unable to reproduce this issue -- your config file seems correct, and linting seems to work fine when I use your config file.

Are you sure your config file has the right name? In the issue you mentioned `eslint.rc` (not sure if this was a typo), but the error message mentions `.eslintrc`.

**Comment by artin-phares** [comment]
Ok, I got it.  
I had two versions of eslint - local (`v4.3.0`) and global (`v3.17.1`)  

Usually I would use local eslint through `gulp` task, but when testing new rules I've used global one.  
Updated global install - now everything is fine.  

As a side node I would suggest adding eslint version into its output.

**Comment by platinumazure** [comment]
@artin-phares That's an excellent idea, would you like to create an issue about adding ESLint version to error outputs?
